worldedit-core/src/main/java/com/sk89q/worldedit/regions/EllipsoidRegion.java

```diff
@@ -80,9 +82,15 @@ public BlockVector3 getMaximumPoint() {
         return center.toVector3().add(getRadius()).toBlockPoint();
     }
 
+
     @Override
-    public int getArea() {
-        return (int) Math.floor((4.0 / 3.0) * Math.PI * radius.getX() * radius.getY() * radius.getZ());
+    public long getVolume() {
+        return BigDecimal.valueOf((4.0 / 3.0) * Math.PI)
```

Comment:
Curious as to why you made `PI` a constant in the other class, but not this computation.

Reply:
I made it a constant because it was its own BigDecimal, this one I could merge it with the other two. Although this is still not going to change so I could pull this one out too tbh.

Reply:
Yea, that's what I was thinking, is making `BigDecimal.valueOf(4.0 / 3.0 * Math.PI)` a constant.